easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,85 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = '2.6.0'
+versionsuffix = '-Python-2.7.12'
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
+
+# The patch contains two fixes:
+# - The authors forgot to reinstall the correct root configure script (which is 
+#   available as configure.orig).
+# - A correction to Makefile.in.top, install-toolkit target: The last install
+#   created an error message, so it has been replaced by two commands that 
+#   hopefully perform the action that the authors intended.
+patches = ['%(name)s-%(version)s_general-fixes.patch']
+
+# Loading this version of Boost will also automatically load Python, and the configure
+# of BLAST+ will discover it and enable Python support.
+#dependencies = [ 
+#	('Boost', '1.63.0', versionsuffix),
+#	('GMP', '6.1.1'),
+#	('libpng', '1.6.26'),
+#	('libjpeg-turbo', '1.5.0')
+#]
+dependencies = []
+
+# Select library type
+configopts  = '--with-static '   # Build failed when --with-dll was added.
+# Executables to generate
+configopts += '--with-static-exe --with-bin-release --without-debug --with-strip '
+# Use a couple of build-in libraries rather than their independent
+# counte3rpart, even if they are installed in seperate modules, to reduce dependencies.
+configopts += '--without-z --without-bz2 --without-pcre '
```

Comment:
By doing it this way, I am sure that the versions that are compiled in are also tested by the authors of BLAST+ and known to be compatible. 

Moreover, the extensive loading from modules that is occurring when building software the traditional EasyBuild way is becoming a pain for users who are building workflows. In fact, the recompile started because a user needed software that we have installed in different toolchains that conflict. Though things seemed to work properly when unloading modules from older toolchains before loading packages that were compiled with a newer one, it was a pain due to the high number of modules involved.